root/static/scripts/edit/externalLinks.js

```diff
@@ -437,9 +458,14 @@ export class ExternalLink extends React.Component<LinkProps> {
             value={props.url}
           />
           {props.errorMessage &&
-            <div className="error field-error" data-visible="1">
-              {props.errorMessage}
-            </div>}
+            <>
+              <div className="error field-error" data-visible="1">
+                {props.errorMessage}
+              </div>
+              <div className="error-target">
+                {`(${props.errorTarget})`}
+              </div>
```

Comment:
I'm a bit confused about why we would print this :) Is it just temporary until this is put on top of code that actually has separate error fields for each? :)

Reply:
Yes, it is just a temporary solution.

Reply:
It would better fit as a label by displaying it in front of the error message with a colon `:` as separator.

Reply:
I assume we'll want to merge this so you can base other PRs (adding the separate error fields) on top of it, since basing PRs on top of other PRs is not a good development or code review experience. :)  If so, you'll have to remove this temporary display so that the UI doesn't appear broken or incoherent after we merge it.

Reply:
This temporary display has been replaced with a div class in commit https://github.com/metabrainz/musicbrainz-server/pull/2128/commits/ad4fcbeb73f1e8bc9be965f222beed21b85761da.